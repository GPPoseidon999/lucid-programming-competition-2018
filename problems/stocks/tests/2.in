887065 539363 587032 152283 875340 319596 754739 241651 170502 527375 652064 383928 687537 4891 657391 398938 880740 742440 405135 752025 50953 125653 356383 956773 914682 751122 167510 856003 367478 703470 192770 286857 405657 870373 301605 799095 914821 464842 926458 668329 601683 831201 912769 663599 440759 327210 49743 234653 557516 392004 206149 46540 63519 960674 118639 455218 21324 277730 915482 956253 5597 721198 694878 799482 901149 456392 510883 544360 546998 548829 452223 666790 30206 877561 575193 334895 477652 885299 496167 730816 111194 700968 990696 397670 610627 359420 436418 436733 236623 811957 704861 377015 884911 213790 56193 940185 46897 721058 741361 276355 889514 29709 655921 989739 218376 875077 237237 691122 340501 885467 434815 137532 407502 241425 830976 139056 308904 997068 490677 528466 264292 10659 967329 222580 13669 115044 246358 355057 332035 265528 280975 442113 372039 153960 887216 280712 742176 900249 299609 85966 423224 90433 97858 998321 344829 135502 529232 48840 308203 541430 720774 788844 987488 580325 404764 97561 404860 722870 270746 422279 681680 728006 601081 962546 679250 418774 522996 870735 826021 214872 554363 143444 964035 504072 160417 735305 781771 560180 834555 552989 670868 291903 71421 156285 642483 979506 124174 280505 121359 440722 106123 892602 309049 697672 162823 305061 661697 100849 43354 13210 404751 747182 686280 443691 489970 422466 725723 426168 28885 670680 513530 933428 814255 504422 167889 11094 430916 478122 981037 218726 320611 428304 121815 16421 354147 306801 383221 410531 475439 1029 744104 378142 770966 589591 597255 754402 406349 46367 814603 569128 677751 987287 330091 496821 642095 535716 880480 471381 102696 126528 303005 725251 768995 706832 952057 753939 495102 472809 229054 181604 152829 536258 553798 78489 631148 301373 715938 829793 726853 949235 440695 707688 943070 572483 712286 733052 37170 320367 812022 259653 359087 606016 992736 94843 316201 76587 753455 39898 106571 79506 988182 586795 205974 55589 428324 156100 689059 689011 91467 153079 44687 967658 133053 777416 893162 948775 358584 686901 291754 575233 712014 23602 217540 332865 113779 934762 106846 499817 378160 466742 918189 239758 365665 796982 561653 489626 481859 288213 750284 343231 267930 937204 680394 507158 285321 186830 392261 916188 465478 396161 371429 996061 502588 30164 705078 180508 701119 17177 956212 879854 889282 461623 568689 582315 303714 999835 760522 401017 948151 629011 806122 21110 193365 347592 487862 786246 828265 969047 769244 809274 440230 338699 496153 730828 18763 296920 147761 561493 309004 229283 92967 803565 961699 951377 674165 750772 748796 315243 939964 354365 58632 854949 585713 615906 254115 84200 788312 443578 686135 868969 363145 769547 360321 446959 260692 158196 893991 204034 566208 404937 133243 239219 326332 125990 170619 303392 729376 538857 411180 530583 274369 413141 224503 330306 90001 358230 519755 544947 459799 495826 501936 996992 306324 16098 146856 299237 882545 51996 938285 532718 732890 923188 289475 820669 779499 754675 755970 475329 273514 978453 284782 691026 685843 748423 934674 674163 662595 532884 695748 329211 714603 339388 457584 747535 663248 638308 479924 682087 579606 358519 671931 405036 37389 639611 934134 382049 116243 749442 247845 307027 333457 914862 927809 920925 880968 87082 83585 856678 941267 70945 966044 859794 796512 563052 414051 563241 294042 927509 50450 25778 258625 696712 811584 304566 908966 935186 542818 231285 917327 585887 528081 121427 425788 924664 313 801495 699915 783908 425807 288794 920555 395884 67267 680491 689706 424312 308489 11397 748810 164970 287937 702968 218575 182416 118640 543007 81447 150651 598390 676414 171046 818384 587851 111561 740048 434741 963367 498043 53339 857151 804293 521084 688054 929295 387108 779308 644397 656635 322318 170469 168548 103993 268216 546844 190247 465695 421930 635206 462087 192554 752957 299045 199771 157720 502890 766038 830070 975829 619571 808691 547718 810157 462943 102944 892690 741155 892676