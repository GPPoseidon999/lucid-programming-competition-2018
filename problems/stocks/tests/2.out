998806
996800
994065
998638
998211
999024
989873
997355
995061
-99